docs/cli/cli.md

```diff
@@ -35,6 +35,8 @@ Consider the following before setting up the integration using the Pulse CLI:
 
     Although the field `system` is optional, if you don't report this information you won't be able to filter the data on the Pulse dashboards. Events without a specified `system` will be grouped under a special system called `_unknown_`.
 
+-   When reporting deployments to Pulse you should use the field `teams` to associate each deployment with the list of teams that contributed with changes included in the deployment.
```

Comment:
In practice, I'm guessing how things will work on the CLI for now. :slightly_smiling_face:

Reply:
You're right. The API/CLI endpoint documentation will state the following:

> "List of teams that are responsible for the changes in this deployment."